server/backend/api/app/models/model.py

```diff
@@ -715,7 +715,7 @@ class FamUserRoleXref(Base):
     create_date = Column(
         TIMESTAMP(timezone=True, precision=6),
         nullable=False,
-        default=datetime.datetime.now(datetime.UTC),
+        server_default=func.utcnow(),
```

Comment:
Use `func.utcnow()` instead of `func.utcnow()` to prevent using "fixed" time during model loading. Will create a separate ticket for dealing with all changes in other models.